hive-metastore/src/test/java/org/apache/iceberg/hive/HiveTableTest.java

```diff
@@ -213,7 +213,7 @@ public void testDropTable() throws IOException {
     table.newAppend().appendFile(file1).appendFile(file2).commit();
 
     // delete file2
-    table.newDelete().deleteFile(file2.path()).commit();
+    table.newDelete().deleteFile(file2).commit();
```

Comment:
Is this change required? There should be no behavior change, right?

Reply:
it's not a required change, but I wanted to move existing places to the non-path-based delete path